frontend/public/components/utils/resource-log.jsx

```diff
@@ -396,6 +417,32 @@ class ResourceLog_ extends React.Component {
             actionLinks={<AlertActionLink onClick={this._restartStream}>Retry</AlertActionLink>}
           />
         )}
+        {hasTruncated && (
+          <Alert
+            isInline
+            className="co-alert"
+            variant="warning"
+            title={'Some lines have been abridged because they are exceptionally long.'}
+          >
+            To view unabridged log content, you can either{' '}
+            <Button
+              component="a"
+              variant="link"
+              isInline
+              href={logURL}
+              target="_blank"
+              noopener
+              noreferrer
+            >
+              open the raw file in another window
+            </Button>{' '}
+            or{' '}
+            <Button component="a" variant="link" isInline href={logURL} download>
```

Comment:
```suggestion
            <Button variant="link" isInline href={logURL} download>
```

Reply:
Changed to a link since href/download props don't work on button.